tests/autoyast/installation.pm

```diff
@@ -267,6 +268,13 @@ sub run {
             wait_screen_change { send_key 'alt-u' };
             next;
         }
+	elsif (match_has_tag('salt-formula-motd-setup')) {
+	    # used for salt-formulas
+	    send_key $cmd{next};
+	    send_key_until_needlematch('salt-formulas-running-provisioner', $cmd{next});
+	    wait_screen_change {};
```

Comment:
Looks like a hack, do we actually need this? Or we can handle it with proper needles?

Reply:
xmm. I match the running-provisioner. This will close automatically and go on. But i just noticed something. so let me run a check and i see

Reply:
send_key_until_needlematch was not needed in fact. Something changed.. Maybe was the bug fix (see ticket for more) or the change i did in the xml. what send_key_until_needlematch used for was because of the behavior as you see here http://aquarius.suse.cz/tests/2200#step/installation/24. But now after the  running-provisioner you dont go to http://aquarius.suse.cz/tests/2200#step/installation/25 but it just keep rolling in the "performing Installation"